package/utils/audit/Makefile

```diff
@@ -6,14 +6,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=audit-userspace
-PKG_VERSION:=3.1.4
-PKG_RELEASE:=2
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://github.com/linux-audit/audit-userspace/archive/refs/tags/v$(PKG_VERSION).tar.gz?
-PKG_HASH:=aec501760acd13ebbe00e78b9b59f795d16a430b1d673628e346cd18905c594b
+PKG_VERSION:=3.1.5
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/linux-audit/$(PKG_NAME).git
+PKG_SOURCE_DATE:=2024-07-29
```

Comment:
Please remove the `PKG_SOURCE_DATE` to use the version number in the package only.